engine/crates/engine-v2/schema/src/sources/graphql.rs

```diff
@@ -0,0 +1,156 @@
+use url::Url;
+
+use crate::{HeaderId, HeaderWalker, RequiredFieldSet, RequiredFieldSetId, SchemaWalker, StringId, SubgraphId, UrlId};
+
+#[derive(Default)]
```

Comment:
Pretty much the same a `sources/federation.rs` that was removed, just with renames